From 9ead8c42e9a72b0b05094b67aba145ce3b23fb34 Mon Sep 17 00:00:00 2001
From: Torsten Dreyer <torsten@t3r.de>
Date: Sat, 27 Jan 2018 10:34:53 +0100
Subject: [PATCH] Disable terrafit

---
 src/Prep/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Prep/CMakeLists.txt b/src/Prep/CMakeLists.txt
index f70dbfa..36e14a0 100644
--- a/src/Prep/CMakeLists.txt
+++ b/src/Prep/CMakeLists.txt
@@ -3,5 +3,5 @@ include_directories(${PROJECT_SOURCE_DIR}/src/Lib)
 add_subdirectory(GDALChop)
 add_subdirectory(DemChop)
 add_subdirectory(Terra)
-add_subdirectory(TerraFit)
+#add_subdirectory(TerraFit)
 add_subdirectory(OGRDecode)
-- 
2.13.6

